docs/designers-developers/developers/data/README.md

```diff
@@ -1,11 +0,0 @@
-# Data Module Reference
```

Comment:
Moved to `docs/data/README.md`. It looks like GitHub didn't pick up this rename.